packages/manager/apps/telecom/package.json

```diff
@@ -149,4 +149,4 @@
     "webpack": "^4.41.2",
     "webpack-merge": "^4.2.2"
   }
-}
\ No newline at end of file
+}
```

Comment:
Rebasing this Pull Request on `release/2020-w43` in order to exclude following diff.

Reply:
Rebased successfully on `release/2020-w43`.